sygnal/apnspushkin.py

```diff
@@ -188,6 +194,12 @@ def __init__(self, name: str, sygnal: "Sygnal", config: Dict[str, Any]) -> None:
                 loop=loop,
             )
 
+        push_type = self.get_config("push_type", str)
+        if not push_type:
+            self.push_type = None
+        else:
+            self.push_type = ApnsPushTypeHeader(push_type)
```

Comment:
As far as I understand it, aioapns expects an `aioapns.PushType` (which is an `Enum`) here. So I think something we should do is: first check whether the value of `push_type` is a member of this enum, and then use the matching value within the enum for `self.push_type`.

I think this is best done by declaring a constant which is a dictionary between a string value and its matching enum member (because annoyingly there isn't a way to list all members of an enum in Python without doing some voodoo with its internals). So something like this:

```python
APNS_PUSH_TYPES = {
    "alert": PushType.ALERT,
    "background": PushType.BACKGROUND,
    ...
}
```

(you can see the list of members of `PushType` here: https://github.com/Fatal1ty/aioapns/blob/26c2cdc3aacb529f4de1065bf3d1cff2137f13bc/aioapns/common.py#L9-L15 - unfortunately aioapns doesn't seem to have docs so we need to refer to the code)

Then you can do something like:

```python
push_type = self.get_config("push_type", str)
if not push_type:
    self.push_type = None
else:
    if push_type not in APNS_PUSH_TYPES.keys():
        raise PushkinSetupException(f"Invalid value for push_type: {push_type}")

    self.push_type = APNS_PUSH_TYPES[push_type]
```

Which a) ensures we use a supported value, b) removes the need for an internal class in favour of one provided by aioapns.